packages/ui/resolution/tests/resolution-specs.js

```diff
@@ -124,12 +137,32 @@ describe('Resolution Specs', () => {
 			expect(getScreenType({width: UHD.width + 1, height: UHD.height - 1})).toBe('wqhd');
 			expect(getScreenType({width: UHD.width - 1, height: UHD.height + 1})).toBe('wqhd');
 
+			expect(getScreenType({width: WUHD.width - 1, height: WUHD.height - 1})).toBe('wqhd');
+			expect(getScreenType({width: WUHD.width + 1, height: WUHD.height - 1})).toBe('wqhd');
+
+			expect(getScreenType({width: UHD2.width - 1, height: WUHD.height - 1})).toBe('wqhd');
+			expect(getScreenType({width: UHD2.width + 1, height: WUHD.height - 1})).toBe('wqhd');
+
 			// UHD
 			expect(getScreenType({width: UHD.width + 1, height: UHD.height + 1})).toBe('uhd');
 
-			expect(getScreenType({width: UHD2.width - 1, height: UHD2.height - 1})).toBe('uhd');
-			expect(getScreenType({width: UHD2.width + 1, height: UHD2.height - 1})).toBe('uhd');
-			expect(getScreenType({width: UHD2.width - 1, height: UHD2.height + 1})).toBe('uhd');
+			expect(getScreenType({width: WUHD.width - 1, height: WUHD.height + 1})).toBe('uhd');
+
+			expect(getScreenType({width: WUHD.width - 1, height: UHD2.height - 1})).toBe('uhd');
+			expect(getScreenType({width: WUHD.width - 1, height: UHD2.height + 1})).toBe('uhd');
+
+			// WUHD
+			expect(getScreenType({width: WUHD.width + 1, height: WUHD.height + 1})).toBe('wuhd');
+
+			expect(getScreenType({width: WUHD.width + 1, height: UHD2.height - 1})).toBe('wuhd');
+			expect(getScreenType({width: WUHD.width + 1, height: UHD2.height + 1})).toBe('wuhd');
+
+			expect(getScreenType({width: UHD2.width - 1, height: WUHD.height + 1})).toBe('wuhd');
+			expect(getScreenType({width: UHD2.width + 1, height: WUHD.height + 1})).toBe('wuhd');
```

Comment:
can you put in the PR description why these hybrid scenarios are needed? UHD2 + WUHD dimensions are used